CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+
+6.0.0-beta.5 / YYYY-MM-DD
```

Comment:
I have concerns using unreleased version numbers in CHANGELOG. This can easily lead to faulty version numbers, missing tags, or changes not being released but in the changelog. Especially when you think of a bugfix release, this usually shouldn’t include new features. I have seen too many faulty release workflows that I can approve this as good practice, sorry. I’d like to have all the unreleased changes under the title `HEAD`.